build.sbt

```diff
@@ -28,26 +28,33 @@ buildInfoPackage := "com.jakehschwartz.finatra.swagger"
 buildInfoKeys := Seq[BuildInfoKey](name, version, swaggerUIVersion)
 
 libraryDependencies ++= Seq(
+  "com.fasterxml.jackson.datatype" % "jackson-datatype-joda" % jacksonVersion,
```

Comment:
Did these just get moved around to test? Its probably better to send empty commits in the future

Reply:
These were sorted for readability